Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/) for Kodi add-ons: skins/skin.estouchy.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-skins/-/skin-estouchy/horizontal-auto.svg)

gade01 commented:
This PR should be left open.
This is to test what Weblate changes.
Weblate will push to this PR once every week until merged or closed.

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/
Translation: Kodi add-ons: skins/skin.estouchy
